cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2199,6 +2201,9 @@ function start-kube-controller-manager {
   if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
```

Comment:
can you also remove the nodeipam from the resource partition as well. thanks

Reply:
ok. i'll remove this after rebase

Reply:
fixed.